ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/picklist/dropdownActivityPicklistQuestion.component.ts

```diff
@@ -66,6 +67,17 @@ import { NGXTranslateService } from '../../../services/internationalization/ngxT
         .ddp-dropdown-field ::ng-deep .mat-form-field-infix {
           width: auto;
         }
+        @media only screen and (max-width: 1024px) {
+          ::ng-deep .mat-form-field .mat-select-native-control{
+          line-height: 17px;
+            }
+          ::ng-deep .mat-form-field .mat-select{
+            line-height: 17px;
+            }
+          ::ng-deep .mat-form-field .mat-form-field-infix input{
+            line-height: 17px;
+            }
+        }
```

Comment:
suggest shortening lines 70-80 like
```
@media only screen and (max-width: 1024px) {
    ::ng-deep .mat-form-field {
        .mat-select-native-control, 
        .mat-select, 
        .mat-form-field-infix input{
            line-height: 17px;
        }
    }        
 } 
```

Reply:
I tried to put it in that way. But it did not work properly. I guess it's because we need to use an ampersand ('&') and it's not working since this is CSS embedded into the component.